src/config.rs

```diff
@@ -311,6 +316,10 @@ impl Config {
              .long("allow-dubious-hosts")
              .help("Allow dubious host names in rsync and HTTPS URIs")
         )
+        .arg(Arg::with_name("fresh")
+            .long("fresh")
+            .help("Delete all locally stored data, download everything again") 
```

Comment:
Stupid question: is it possible to pass `--fresh` to an invocation of Routinator that will NOT download anything thus invalidating this description?

Reply:
Yes. This might be worthwhile mentioning in the man page. The help text needs to be short.